base/applications/rapps/unattended.cpp

```diff
@@ -312,6 +313,48 @@ HandleInfoCommand(CAppDB *db, LPWSTR szCommand, int argcLeft, LPWSTR *argvLeft)
     return TRUE;
 }
 
+static unsigned int CALLBACK
+RunGuiThread(void *nCmdShow)
+{
+    WCHAR fakecmdline[] = L"://";
+    return ParseCmdAndExecute(fakecmdline, PCAEF_WINDOWONLY, (UINT)(SIZE_T)nCmdShow);
```

Comment:
```suggestion
    return ParseCmdAndExecute(fakecmdline, PCAEF_WINDOWONLY, PtrToInt(nCmdShow));
```
the ParseCmd.... function takes a signed int for the cmdshow.